Projekty/Projekt1/Kacper_Grzymkowski_Jan_Gaska/Inzynieria Cech, wstępne modelowanie/WUM_Kacper_Grzymkowski_Jan_Gaska_Kamien_Milowy_II.ipynb

```diff
@@ -0,0 +1,2102 @@
+{
```

Comment:
<p>Dobre, że opakowano kod w funkcje. <u>Skoro zbiór nie jest zbalansowany to być może lepiej niż analizować accuracy, byłoby stosować F1-score.</u></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>